prov/efa/src/rxr/rxr_read.h

```diff
@@ -117,7 +117,7 @@ int rxr_read_init_iov(struct rxr_ep *ep,
 
 int rxr_read_post(struct rxr_ep *ep, struct rxr_read_entry *read_entry);
 
-int rxr_read_post_remote_read_or_queue(struct rxr_ep *ep, int entry_type, void *x_entry);
+int rxr_read_post_remote_read_or_queue(struct rxr_ep *ep, struct rxr_op_entry *x_entry);
```

Comment:
for consistency, `x_entry` to `op_entry`

Reply:
Fixed in new revision.